ddl/index.go

```diff
@@ -524,6 +529,10 @@ func (w *addIndexWorker) fetchRowColVals(txn kv.Transaction, taskRange reorgInde
 			}
 
 			w.idxRecords = append(w.idxRecords, idxRecord)
+			if handle == taskRange.endHandle {
```

Comment:
Is this check necessary here?

Reply:
Is It checked in line 516？

Reply:
yes,, we need handleOutOfRange be true, outside will use this boolean.